### PR TITLE
Feature/posts hugo pereira

### PR DESCRIPTION
**Pull request by putWorkDev**

### CODE REVIEW

File: src/main/java/com/hp/posts/controller/PostController.java [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+        return ResponseEntity.ok(postRepository.findAll());
+    }
+
+    @PostMapping("/upvote/{postId}")
+    public ResponseEntity<?> increaseVote(@PathVariable Integer postId) {
+        Optional<PostModel> optionalPostModel = postRepository.findById(postId);
+        if (optionalPostModel.isPresent()) {
+            PostModel postModel = optionalPostModel.get();
+            postModel.setVotesUp(postModel.getVotesUp()+1);
+            postModel.setVotesUpPercentage(postModel.getVotesUp().toString().substring(0, 1).concat("%"));
```
Comment (ruialexsantos) [inline src/main/java/com/hp/posts/controller/PostController.java]: Could you please explain this line(line 34)?

Comment (putWorkDev) [reply src/main/java/com/hp/posts/controller/PostController.java]: i am extracting the first digit of the vote, to build the percentage, as it is asked

Comment (RicCastelhano) [reply src/main/java/com/hp/posts/controller/PostController.java]: Hi Hugo. Humm, it seems you missunderstood the challenge. Check it out once more please:

"Your solution should consider that if you have a first post with 600 up votes and 400 down votes means that you have 60% of up votes and 40% of down votes, and if you have another post with 6 up votes and 4 down votes you have the same % of the previous post, 60% of up votes and 40% of down votes.
Hint: Note that the score in % of the two posts are equal, but the real "values" are significantly differents"

Your solution should be able to distinguish the weight of the total votes for each post, and a post with 1000 votes and 60% upvote is more important than a post with 10 votes with 60% upvote.

Comment (putWorkDev) [reply src/main/java/com/hp/posts/controller/PostController.java]: hi, fixed. if you could gave a look, much thanks

**Pull request closed without merging**